bot/exts/filtering/filtering.py

```diff
@@ -1432,9 +1434,20 @@ async def send_weekly_auto_infraction_report(self, channel: discord.TextChannel
         if len(lines) == 1:
             lines.append("Nothing to show")
 
-        await channel.send("\n\n".join(lines))
-        log.info("Successfully sent auto-infraction report.")
+        report = "\n\n".join(lines)
+        try:
+            await channel.send(report)
+        except discord.HTTPException as e:
+            if e.code == 50035:  # Content too long
+                report = discord.utils.remove_markdown(report)
+                paste_url = await send_to_paste_service(report, extension="txt")
+                file_buffer = io.StringIO(report)
+                await channel.send(
+                    f"**{lines[0]}**\n\n{paste_url}",
+                    file=discord.File(file_buffer, "last_weeks_autoban_filters.txt"),
+                )
```

Comment:
Doesn't this suppress any other reasons for HTTP exceptions? you'll end up not having a report and then logging that it was sent successfully.

Reply:
True, I've made is so that these still get raised.